tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_is_numeric_array
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data__wp_check_existing_file_names
+	 *
+	 * @ticket 53971
+	 *
+	 * @param string $filename filename looked for.
+	 * @param array $files files check against.
+	 * @param boolean $expected Expected result.
```

Comment:
```suggestion
	 * @param string  $filename The file name to check.
	 * @param array   $files    An array of existing files in the directory.
	 * @param boolean $expected Expected result.
```
- Docs alignment adjusted for consistency and readability.
- Parameter descriptions updated to match `_wp_check_existing_file_names()`.